Jenkinsfile

```
@@ -374,7 +361,7 @@ pipeline {
             }
           }
           steps {
-            makeTarget("Metricbeat oss Mac OS X", "TEST_ENVIRONMENT=0 -C metricbeat testsuite")
+            mageTarget("Metricbeat oss Mac OS X", "metricbeat", "unitTest")
```

Nit. `Metricbeat OSS` used in other targets.
```suggestion
            mageTarget("Metricbeat OSS Mac OS X", "metricbeat", "unitTest")
```

---

Fixed. We fix all the naming to be consistent when we implement all the targets based on this spreadsheet https://github.com/elastic/beats/issues/17411#issuecomment-614858689 using a matrix.